Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,58 @@ click: function () { closeClicked.raiseEvent(this); }');
         closeElement.innerHTML = '&times;';
         infoElement.appendChild(closeElement);
 
-        var infoBodyElement = document.createElement('div');
-        infoBodyElement.className = 'cesium-infoBox-body';
-        infoElement.appendChild(infoBodyElement);
-
-        var descriptionElement = document.createElement('div');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('allowfullscreen', true);
+        infoElement.appendChild(frame);
 
         var viewModel = new InfoBoxViewModel();
-        this._viewModel = viewModel;
+        knockout.applyBindings(viewModel, infoElement);
+
+        //We inject default css into the content iframe,
+        //end users can remove it or add their own via the exposed frame property.
+        var cssLink = document.createElement("link");
+        cssLink.href = buildModuleUrl('Widgets/InfoBox/InfoBoxDescription.css');
+        cssLink.rel = "stylesheet";
+        cssLink.type = "text/css";
 
-        knockout.applyBindings(this._viewModel, infoElement);
+        //div to use for description content.
+        var frameContent = document.createElement("div");
+        frameContent.className = 'cesium-infoBox-description';
+
+        this._container = container;
+        this._element = infoElement;
+        this._frame = frame;
+        this._viewModel = viewModel;
+        this._descriptionSubscription = undefined;
+
+        var that = this;
+
+        //We can't actually add anything into the frame until the load event is fired
+        frame.addEventListener('load', function() {
+            var frameDocument = frame.contentDocument;
+            frameDocument.head.appendChild(cssLink);
+            frameDocument.body.appendChild(frameContent);
+
+            //Once the frame is initialized, we can enable sandboxing.
+            frame.setAttribute('sandbox', 'allow-popups allow-forms'); // allow-same-origin allow-pointer-lock allow-scripts allow-top-navigation
```

Comment:
Okay, I made the changes.  I find it odd that you say no sandboxing was being applied though, because you can definitely change `sandbox` after the fact without any issues, I was testing it last night.  Either way, this should be good to go now.

I know @shunter's idea, but I'd rather go with this for now and we can experiment more and try other stuff if we run into issues.